packages/frontend-v2/pages/index.tsx

```diff
@@ -33,16 +38,29 @@ const Home: NextPage = () => {
     );
   };
 
+  const clearDataBase = async () => {
+    const res = await axios.get("http://localhost:3002/api/students");
+    const uuids = res.data.map((student: GetStudentResponse) => student.uuid);
+    for (const uuid of uuids) {
+      await axios.delete(`http://localhost:3002/api/students/${uuid}`);
+    }
```

Comment:
this is slow, because it doesn't make parallel requests. instead do something like 

```typescript
await Promise.all(uuids.map(uuid => axios.delete(...)));
```